src/fileActions.jsx

```diff
@@ -310,6 +310,10 @@ const EditPermissionsModal = ({ selected, path }) => {
         ];
     }
 
+    function sortByName(a, b) {
+        return a.name.toLowerCase() < b.name.toLowerCase() ? -1 : 1;
```

Comment:
Let's use `a.name.localeCompare(b.name)`?

Reply:
Nice one, in that case, we should also update the sorting in the files card body